fe/src/main/java/org/apache/doris/load/loadv2/dpp/DppUtils.java

```diff
@@ -0,0 +1,230 @@
+            return null;
+        }
+        if (dataType.equals(DataTypes.BooleanType)) {
+            return Boolean.class;
+        } else if (dataType.equals(DataTypes.ShortType)) {
+            return Short.class;
+        } else if (dataType.equals(DataTypes.IntegerType)) {
+            return Integer.class;
+        } else if (dataType.equals(DataTypes.LongType)) {
+            return Long.class;
+        } else if (dataType.equals(DataTypes.FloatType)) {
+            return Float.class;
+        } else if (dataType.equals(DataTypes.DoubleType)) {
+            return Double.class;
+        } else if (dataType.equals(DataTypes.DateType)) {
+            return Date.class;
+        } else if (dataType.equals(DataTypes.StringType)) {
+            return String.class;
+        }
+        return null;
+    }
+
+    public static Class columnTypeToClass(String columnType) {
+        switch (columnType) {
+            case "BOOL":
+                return Boolean.class;
+            case "TINYINT":
+            case "SMALLINT":
+                return Short.class;
+            case "INT":
+                return Integer.class;
+            case "DATETIME":
+            case "BIGINT":
+                return Long.class;
+            case "LARGEINT":
+                return BigInteger.class;
+            case "FLOAT":
+                return Float.class;
+            case "DOUBLE":
+                return Double.class;
+            case "DATE":
+                return Date.class;
+            case "HLL":
+            case "CHAR":
+            case "VARCHAR":
+            case "BITMAP":
+            case "OBJECT":
+                return String.class;
+            default:
+                return String.class;
+        }
+    }
+
+    public static DataType columnTypeToDataType(String columnType) throws UserException {
+        DataType dataType = DataTypes.StringType;
+        switch (columnType) {
+            case "BOOL":
+                dataType = DataTypes.BooleanType;
+                break;
+            case "TINYINT":
+            case "SMALLINT":
+                dataType = DataTypes.ShortType;
+                break;
+            case "INT":
+                dataType = DataTypes.IntegerType;
+                break;
+            case "DATETIME":
+            case "BIGINT":
+            case "LARGEINT":
+                // todo: special treat LARGEINT because spark do not support int128
+                dataType = DataTypes.LongType;
+                break;
+            case "FLOAT":
+                dataType = DataTypes.FloatType;
+                break;
+            case "DOUBLE":
+                dataType = DataTypes.DoubleType;
+                break;
+            case "DATE":
+                dataType = DataTypes.DateType;
+                break;
+            case "HLL":
+            case "CHAR":
+            case "VARCHAR":
+            case "BITMAP":
+            case "OBJECT":
+                dataType = DataTypes.StringType;
```

Comment:
1. For HLL and BITMAP,output schema is Binary instead of String
2. A better understand method name is,```columnTypeToClass ForInputSchema``` and ```columnTypeToDataType for output schema```

Reply:
the input type for hll and bitmap is string.